cluster-provision/okd/scripts/provision.sh

```diff
@@ -249,15 +250,18 @@ until oc -n openshift-cluster-version scale --replicas=0 deploy cluster-version-
     sleep 5
 done
 
+
 # Scale down machine-config-operator to prevent re-creation of master machineconfigpools
 until oc -n openshift-machine-config-operator scale --replicas=0 deploy machine-config-operator; do
```

Comment:
maybe there are other lines that needs to be deleted like this one. wasn't sure.